### PR TITLE
Add "fill level" sprites to mops and damp rag

### PR DESCRIPTION
**Pull request by Tayrtahn**
## About the PR

Adds fill level sprites to the mop, advanced mop, and damp rag. This means that they now change appearance when holding liquids, including taking on the color of the solution.

## Why / Balance

Makes it a bit easier for janitors/bartenders to see when their mops/rags are dirty and shows what they've been mopping. Immersion!

## Technical details

The ability for fill sprites to work with wielded items was added by #28532, so this is just sprites and YAML.

## Media

https://github.com/space-wizards/space-station-14/assets/85356/8931664e-0c1c-4453-a498-6519aaf2a89b

Screenshot 2024-06-01 at 8 19 04 PM
Screenshot 2024-06-01 at 8 19 42 PM
Screenshot 2024-06-02 at 1 48 14 PM

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Mops and rags now show the liquids they soak up.

### CODE REVIEW

**Comment by beck-thompson** [comment]
Such a good idea! 👍

**Comment by lzk228** [comment]
its too blue with the water

**Review by AJCM-git — COMMENTED** [review]
i think it would be better to just rename the SolutionContainerLayers.Fill state in the rsi to a common name  and then have the SolutionContainerVisuals component assume fill-maxFillLevels is the sprite they want unless overwritten, that way you dont have to copypaste as much in the prototypes

**Comment by Tayrtahn** [comment]
> i think it would be better to just rename the SolutionContainerLayers.Fill state in the rsi to a common name and then have the SolutionContainerVisuals component assume fill-maxFillLevels is the sprite they want unless overwritten, that way you dont have to copypaste as much in the prototypes

So people don't need to specify "fill-" and "-fill-", it just uses the one for both? That sounds reasonable.

**Comment by AJCM-git** [comment]
yeah, basically to bypass having to overwrite the layers

**Comment by Tayrtahn** [comment]
Sounds good, but it'll require a lot of cleanup work with existing content. Separate PR?

**Comment by AJCM-git** [comment]
> Sounds good, but it'll require a lot of cleanup work with existing content. Separate PR?

yeah, thats fair

File: Resources/Textures/Objects/Specific/Janitorial/advmop.rsi/meta.json
[the remark this answers is not in the document]
Comment (Tayrtahn) [reply Resources/Textures/Objects/Specific/Janitorial/advmop.rsi/meta.json:1]: Thank you!

**Pull request merged**